.github/workflows/ci-test.yml

```diff
@@ -162,9 +162,13 @@ jobs:
           key: crates-${{ runner.os }}-${{ hashFiles('Cargo.lock') }}
           restore-keys: "crates-${{ runner.os }}"
       - name: run unit tests
-        run: $pre_command && cargo x test --jobs ${max_threads} --unit --changed-since "origin/$TARGET_BRANCH"
+        run: |
+          $pre_command && cargo x test --jobs ${max_threads} --unit --changed-since "origin/$TARGET_BRANCH"
+          sccache -s
         env:
           TARGET_BRANCH: ${{ needs.prepare.outputs.changes-target-branch }}
+          SCCACHE_AWS_ACCESS_KEY_ID: ${{ secrets.SCCACHE_AWS_ACCESS_KEY_ID }}
```

Comment:
for the push to auto branch, let's populate sccache at this time.   there may be some wasted pushes of broken items, but aside from a little wasted cache space, not a big deal.